### DISCUSSION TITLE
Can not submit competition result

### DISCUSSION THREAD

**Discussion by CarlHuangNuc** [bdd100k/bdd100k#244] [Q&A]
Hi, Hoster & Orginazer,

Why currently we can not submit the CVPR2022 BDD MOT Competition Result?

When evaluate Server can normal work ?

Hope some people can give some comments or feadback.

**Comment by siyuanliii** [comment]
Can you give more details of your error?

**Reply by CarlHuangNuc** [reply]
![image](https://user-images.githubusercontent.com/104048874/170276587-7aed325b-bb63-41ba-97bd-5689dc1f2829.png)

**Comment by CarlHuangNuc** [comment]
current we submit result, get all Nones , on CVPR2022 BDD MOT Challenge; even  the result has worked previously, and re-submit, all None Value.

from this platform, several people talk about submit None issue ..

**Reply by siyuanliii** [reply]
The " None" means your job hasn't started. I see you have successful submissions. Yes, sometimes the evaluation server on eval ai is not stable due to their recent update. It also happens with other challenges. It usually recovers after a while. Since the server is not maintained by us, If you need the results in a hurry, you could email the eval ai support: Ram( ramramrakhya81@gmail.com. ) He will restart the evaluation for you.

**Reply by siyuanliii** [reply]
Yes, I will write to the eval ai support regarding this issue.

**Reply by fabianwindbacher** [reply]
Just to add on here, I have the same issue with the Pose Estimation challenge.
It worked for me in the past; and then it did not (just like in the screenshot with all Nones).

I tried it 2 times 1 week apart (May 16 & May 23); both times it failed.

**Reply by CarlHuangNuc** [reply]
> Yes, I will write to the eval ai support regarding this issue.

thanks, Eval AI can normal work and receive our submited result.
